script.js

```diff
@@ -0,0 +1,137 @@
+//=======GLOBAL VARIABLES=======// 
+
+const apiKey = "f7aa1eb0bfab45cdae5e9f2f21292092"
+const container = document.getElementById("container")
+const buttons = document.querySelectorAll(".btn-kitchen")
+const sortButtons = document.querySelectorAll(".btn-sorting")
+const randomButton = document.querySelector(".btn-random")
+
+let selectedCuisine = "All"
+let currentRecipes = []
+
+//======= FUNCTIONS for UI =======//
+
+//Buttons, changes color when active//
+function makesButtonInteractive(groupClass) {
```

Comment:
Either stick to functions declared like:
`function myFunctionName () {}`
or 
`const myFunctionName = () => {}`
don't mix